ts/utils/input.ts

```diff
@@ -128,3 +133,15 @@ export function getCreditCardFromState(
 
   return some(card);
 }
+
+/**
+ * This function checks if the cardholder charset is ammitted.
```

Comment:
```suggestion
 * This function checks if the cardholder charset is admitted.
```

Reply:
Solved in [707775a](https://github.com/pagopa/io-app/pull/2833/commits/707775af8cdbd6bc7afb627deb243280df192968)